app/models/concerns/project_media_creators.rb

```diff
@@ -73,9 +73,8 @@ def set_quote_metadata
 
   protected
 
-  def create_video_or_image
-    extname = File.extname(self.file.path).delete('.')
-    m = VideoUploader.upload_extensions.include?(extname) ? UploadedVideo.new : UploadedImage.new
+  def create_video_or_image(media_type)
```

Comment:
Sawy, in order to guarantee backward compatibility, I think we should default `media_type` to `UploadedImage`

Reply:
Done